### PR TITLE
[Feat #3]: Created static home page

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
- this is a static home page
- no functionality

### Screenshot
Screenshot (195)

### CODE REVIEW

**Review by jeetbhalani15 — APPROVED** [review]
- The UI looks good.
- CSS looks good with no repeatation.
- Clean folder name

File: src/components/Checkbox/Checkbox.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./Checkbox.css";
+
+function Checkbox({ title, checked, changeHandler }) {
+  return (
+    <label className="checkbox py-1">
+      <input type="checkbox" checked={checked} onChange={changeHandler} />{" "}
```
Comment (hrishikeshmane) [inline src/components/Checkbox/Checkbox.jsx:7]: remove {" "}  from code

File: src/components/Note/Note.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./Note.css";
+
+function Note({ title, note, color }) {
+  return (
+    <div class={`card m-1 p-2 note-color-${color}`}>
```
Comment (hrishikeshmane) [inline src/components/Note/Note.jsx:6]: rename to className

**Review by hrishikeshmane — APPROVED** [review]

File: src/components/Note/Note.jsx (hunk 2 above)
Comment (SanketDhabarde) [reply src/components/Note/Note.jsx:6]: Thanks for pointing it out. I have removed it in next pr

File: src/components/Checkbox/Checkbox.jsx (hunk 1 above)
Comment (SanketDhabarde) [reply src/components/Checkbox/Checkbox.jsx:7]: It's just to give space between checkbox and label. but I will try other way thanks

**Pull request merged**